src/sap/fhir/model/r4/lib/FHIRRequestor.js

```diff
@@ -232,10 +231,10 @@ sap.ui.define([
 		var sRequestUrl;
 		var sContentType = "application/json";
 		var oBindingInfo = this.oModel.getBindingInfo(oFHIRUrl.getRelativeUrlWithoutQueryParameters());
-		// if the method is GET , secure search is enabled
+		// if the method is GET, secure search is enabled
 		// convert the path to _search and all the url paramters will be converted to POST form data
-	    // except ValueSet/$expand or specific operations  like Patient/53/_history
-		if (sMethod == HTTPMethod.GET && this.bSecureSearch && oFHIRUrl.isSearchAtBaseLevel() && !this._isCsrfTokenRequest()) {
+		// except ValueSet/$expand or specific operations  like Patient/53/_history
+		if (this._canRequestBeTransformed(sMethod, oFHIRUrl)) {
```

Comment:
what do you think about the name `_isRequestTransformable`?

Reply:
sounds good :)

Reply:
done